packages/widget/src/index.tsx

```diff
@@ -17,23 +17,34 @@ import { CloseIcon, ChatIcon, AirplaneIcon, DownloadIcon } from "./icons";
 import * as constants from "./ui/constants";
 import { Props } from "./props";
 import * as C from "./ui/components";
+import { ReactHandler } from "./utils";
 
 export { Props } from "./props";
 
 export const standalone = (
   props: Props
 ): {
   teardown: () => void;
+  expand: () => void;
+  collapse: () => void;
 } => {
+  const reactHandler = new ReactHandler<Props>(Widget);
   const node = document.createElement("div");
   node.setAttribute("id", "widget-container");
   node.setAttribute("style", `z-index: ${constants.largeZIndex};`);
   document.body.appendChild(node);
-  render(<Widget {...props} />, node);
+  reactHandler.mount(node, props);
+
   return {
     teardown: () => {
       unmountComponentAtNode(node);
     },
+    expand: () => {
+      reactHandler.updateProps({ ...props, initiallyExpanded: true });
```

Comment:
The `initiallyExpanded` prop is intended to control whether the widget is expanded at the very beginning at the time it is mounted.